theano/scalar/basic.py

```diff
@@ -3099,7 +3109,8 @@ def init_name(self):
                 for i, r in enumerate(self.fgraph.variables):
                     if r not in io and len(r.clients) > 1:
                         r.name = 't%i' % i
-                rval = "Composite{%s}" % str(self.fgraph)
+                rval = "Composite{%s}" % ', '.join([pprint(output) for output
+                                                    in self.fgraph.outputs])
```

Comment:
@nouiz , I'm not sure that I understand your comment.
But, when I write ```pprint(self.fgraph.outputs)``` I get this error:
```python
AttributeError: 'list' object has no attribute 'owner'
```
By "ugly output", what exactly you mean?